src/app/blog/index.html

```diff
@@ -4,6 +4,34 @@
   windowTitle="SKY UX Blog">
 
 	<article class="blog-post">
+		<header>
+			<h2 class="sky-section-heading">Design update — July 2019</h2>
+		</header>
+		<section>
+			<h3 class="sky-subsection-heading">Recently completed</h3>
+				<h4 class="sky-emphasized"><a routerLink="/components/toggle-switch">Toggle switch</a></h4>
+					<p>We added a toggle switch input for scenarios when users need to change a status outside the context of a larger form.</p>
```

Comment:
"Added a toggle switch input..."   (Following the pattern from previous entries)